utils/data.ts

```diff
@@ -66,7 +66,7 @@ export const transactionTabs: TransactionData = {
       lazlo: 'pending',
     },
     input: {
-      sats: '101,000',
+      sats: '100,000',
     },
     output_0: {
       sats: '100000',
```

Comment:
Does this code validate answers? If the input for the refund tx is 100,000 sats, and you take away 1,000 for miner fees, the output should be 99,000 sats

Reply:
fixed in https://github.com/saving-satoshi/saving-satoshi/commit/788751e65ddc028de04bf8d084fbd12a9c3f5890